src/app/home/home.component.html

```diff
@@ -2,7 +2,6 @@
   <section>
     <header>
       <h2>Your Series</h2>
-      <a class="create" [routerLink]="['/series/new']">New Series &raquo;</a>
```

Comment:
This link was unnecessary and ugly, as there's already a "Create Series" link in the header.